core/src/main/java/com/google/zxing/qrcode/encoder/MatrixUtil.java

```diff
@@ -271,12 +271,7 @@ static void embedDataBits(BitArray dataBits, int maskPattern, ByteMatrix matrix)
   // - findMSBSet(1) => 1
   // - findMSBSet(255) => 8
   static int findMSBSet(int value) {
-    int numDigits = 0;
-    while (value != 0) {
-      value >>>= 1;
-      ++numDigits;
-    }
-    return numDigits;
+    return 32 - Integer.numberOfLeadingZeros(value);
```

Comment:
I know for a fact that the two functions are in Java 5. I checked the Android API now as per your suggestion, and it looks fine too: https://developer.android.com/reference/java/lang/Integer.html#bitCount%28int%29 (API level 1, Android platform 1.0). =)